src/lib_gui/qt/element/QtTabBar.cpp

```diff
@@ -1,4 +1,8 @@
 #include "QtTabBar.h"
+#include <QtContextMenu.h>
+#include <QtGraphicsView.h>
+#include <logging.h>
+#include <QContextMenuEvent>
```

Comment:
Please follow convention:
For Qt headers we don't add '.h' e.g. `#include <QContextMenuEvent>`
For own headers we use double quotations e.g. `#include "QtContextMenu.h"`
Qt headers are placed above own headers (see other files)